### ISSUE TITLE
FTB Omnia 1.15.2 Piper spawning in completely lit and closed in areas

### ISSUE THREAD

**Issue by Saenaria** [Alex-the-666/Rats#317]
I went afk in my completely closed in base that does not have a single spot in it that is a viable spawn for any hostile mob and I came back to being dead with three pipers and 30 rats inside my base?!?!?

**Comment by JakeMoe** [comment]
I've got this in a custom pack I'm making for 1.15.2.  The main part of my base is underground at y=50, only entrance is a 2x2 shaft with ladders.  As an example, while I'm digging out a new room, I might be suddenly attacked by rats and turn around to find a piper there.  Light Overlay mod shows me everything is lit up.  It does appear to be obeying distance rules; it never seems to spawn close to me.  I've also had it happen above ground; I'm on a peninsula that I've walled off and torched completely.  However, it does only happen at night on the surface, so it seems it's obeying sunlight levels, just not block light levels?
